tasks.py

```diff
@@ -29,10 +39,24 @@ def install_minimum(c):
             line = line.strip()
             if line.startswith('#'): # ignore comment
                 continue
+
+            if 'python_version' in line:
```

Comment:
Can you add a comment here to describe the function of these few lines of code?  It seems in the `setup.py` file, only the variable `python_requires` talks python version.

https://github.com/sintel-dev/Orion/blob/master/setup.py#L94
We can add python 3.8 here as well.

Reply:
I added a comment for the code.

@dyuliu what do you mean by
>It seems in the setup.py file, only the variable `python_requires` talks python version.

Reply:
@sarahmish that means in `setup.py` file, there is no line containing the string `python_version`.  The only place mentioning python version is in this variable `python_requires`.